public/profile.css

```
@@ -0,0 +1,72 @@
+@charset "utf-8";
+
+body {
+  display: flex;
+  font-family: sans-serif;
+  justify-content: center;
+  margin:0;
+  background-color: #ffffff;
+  align-items: center;
+  min-height: 100vh;    /* ←画面の高さいっぱいに中央寄せ */
+  
+
+}
+.card {
+  display: flex;
+  gap:32px;
+  background-color: #f0f8ff;
+  border:5px solid #ccc;
+  border-radius: 12px;
+  padding: 24px;
+  margin:40px;
+  background-image: url("background.jpg");
+}
+
+.right {
+  margin-top: 80px;
+}
+
+.tbl {
+  border-collapse: collapse;
+  margin-bottom: 24px;
+  background-color: rgba(255, 255, 255, 0.6); /* 白＋60%透明 */
+}
+
+.tbl td,
+.tbl th {
+  border: 1px dotted #000;         
+  padding: 8px;                    
+  text-align: left; 
+}
```

```suggestion
}

```
スタイルの間に空白行があると見やすいかなと思います！